docs/advanced/monitoring.md

```diff
@@ -33,6 +33,11 @@ the `@WithSpan` annotation in various places to make the trace more readable. Yo
 tracing collector of choice. Our example [docker-compose](../../docker/monitoring/docker-compose.monitoring.yml) deploys
 an instance of Jaeger with port 16686. The traces should be available at http://localhost:16686.
 
+### Configuration Note
+
+We recommend using either `grpc` or `http/protobuf`, configured using `OTEL_EXPORTER_OTLP_PROTOCOL`. Avoid using `http` will not work as expected due to the size of
```

Comment:
Is the http note applicable even with the env vars specified?
```
OTEL_EXPORTER_OTLP_MAX_PAYLOAD_SIZE=4194304 
OTEL_EXPORTER_OTLP_HTTP_HTTP2_MAX_FRAME_SIZE=16777215 
```

Reply:
the first one is, the second one is applicable for other instances besides GMS or MCE which have large traces. No harm in including it.